### ISSUE TITLE
Items covered by the AppBar should be excluded from semantics

### ISSUE THREAD

**Issue by goderbauer** [flutter/flutter#11664]
If you have a List with a floating AppBar, the list items below the app bar are still accessible via a11y means. They should be excluded.

Similarly, when the last visible list item is focused and the user moves the focus to the previous item covered by the app bar, the list should scroll to ensure that the newly focused item is visible on screen (this should happen automatically once the list items hidden behind the app bar are no longer part of the semantics tree).

**Comment by goderbauer** [comment]
This bug of course can be generalized to: All items within a sliver currently covered by another sliver should be excluded.

**Comment by goderbauer** [comment]
This has been fixed by https://github.com/flutter/flutter/pull/11878.
